projects/gui/src/mainwindow.cpp

```diff
@@ -341,9 +341,9 @@ void MainWindow::setCurrentGame(const TabData& gameData)
 	if (gameData.game == m_game && m_game != 0)
 		return;
 
-	for (int i = 0; i < 2; i++)
+	for (auto & m_player : m_players)
 	{
-		ChessPlayer* player(m_players[i]);
+		ChessPlayer* player(m_player);
```

Comment:
This line could be removed completely now, and the "for" statement could just be `for (auto & player : m_players)`